src/ServerRequestFactory.php

```diff
@@ -472,7 +510,7 @@ private static function normalizeNestedFileSpec(array $files = [])
      * @param array $server
      * @return string
      */
-    private static function marshalProtocolVersion(array $server)
+    public static function marshalProtocolVersion(array $server)
```

Comment:
To be reverted - this is `private` API for which we shouldn't provide BC guarantees